index.bs

```diff
@@ -3690,6 +3717,7 @@ Verifiers may use the following algorithm to verify an encoded {{CollectedClient
     1. A string, |type|, that contains the expected {{CollectedClientData/type}}.
     1. A byte string, |challenge|, that contains the challenge byte string that was given in the {{PublicKeyCredentialRequestOptions}} or {{PublicKeyCredentialCreationOptions}}.
     1. A string, |origin|, that contains the expected {{CollectedClientData/origin}} that issued the request to the user agent.
+    1. An optional string, |topOrigin|, that contains the expected {{CollectedClientData/topOrigin}} of the frame that issued the request to the user agent.
```

Comment:
Again, this needs to go after `crossOrigin`.

Reply:
Fixed.